contrib/pkg/deprovision/azure.go

```diff
@@ -1,11 +1,15 @@
 package deprovision
 
 import (
+	"os"
+
 	azuresession "github.com/openshift/installer/pkg/asset/installconfig/azure"
```

Comment:
While cleaning up imports, the "github.com/openshift" imports that are not hive should be in their own section before the "github.com/openshift/hive" imports.